packages/mermaid/src/docs/config/directives.md

```diff
@@ -26,9 +26,9 @@ Mermaid basically supports two types of configuration options to be overridden b
 
 **NOTE:** These options listed here are not all the configuration options. To get hold of all the configuration options, please refer to the [defaultConfig.ts](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/defaultConfig.ts) in the source code.
 
-```
-Soon we plan to publish a complete list of top-level configurations & all the diagram specific configurations, with their possible values in the docs
-```
+::: info
+We plan to publish a complete list of top-level configurations & all the diagram specific configurations, with their possible values in the docs soon.
+:::
```

Comment:
We use ` ```note` as `::: info` isn't valid markdown.

It will be auto-transformed into appropriate format based on the docs generation call.

Reply:
Thanks for reviewing @sidharthv96 !

We get an error message if the backtick blocks don't specify a language (e.g.html, CSS, markdown, etc) as in the screenshot below.  This is causing the text overflow.

![Screen Shot 2022-12-15 at 12 27 06 PM](https://user-images.githubusercontent.com/35910788/207960453-7ac54b45-8b85-419e-8f42-1943c929c7c4.png)

Vitepress allows for [markdown custom containers](https://vitepress.vuejs.org/guide/markdown#custom-containers) (e.g. info, tip, warning, etc.)

![Screen Shot 2022-12-15 at 12 12 34 PM](https://user-images.githubusercontent.com/35910788/207958791-df2b41ab-932c-4c03-b432-94045b7e2b8d.png)

Reply:
An alternative would be to use the left angle bracket before the text.

![Screen Shot 2022-12-15 at 12 42 28 PM](https://user-images.githubusercontent.com/35910788/207963066-33b00383-c970-4f2a-9141-5161595292f8.png)

Reply:
Ohh, I thought we had added support for the vitepress blocks when rendering to vitepress. 

Do you want to give a shot at editing this block based on the `vitepress` flag?
https://github.com/mermaid-js/mermaid/blob/33f78a54294e92898d35edc5d76664a8e1e50115/packages/mermaid/src/docs.mts#L196-L200

We should be rendering the right `::: info` code to vitepress.

Reply:
Yes, will do - thank you @sidharthv96 !

Reply:
This is the output for the updates:

![Screen Shot 2022-12-19 at 8 36 56 PM](https://user-images.githubusercontent.com/35910788/208729022-28b1dfb2-22af-4f65-a63e-5a648a90371c.png)

![Screen Shot 2022-12-19 at 8 35 33 PM](https://user-images.githubusercontent.com/35910788/208729018-10b17d23-ca00-4438-934b-e9aa3f800c8e.png)

![Screen Shot 2022-12-19 at 8 37 39 PM](https://user-images.githubusercontent.com/35910788/208729027-dc135f0e-4de4-451c-b6cf-33f7071c4af1.png)

Reply:
This is wonderful!
Nice touch with the custom icons :)